src/Makefile.am

```diff
@@ -196,4 +196,4 @@ libsub_root_a_SOURCES = \
 	thread_worker.h
 
 
-AM_CPPFLAGS = -I$(srcdir) -I$(top_srcdir)
+AM_CPPFLAGS = -I$(srcdir) -I$(top_srcdir) -DPACKAGE_DATADIR=\"$(pkgdatadir)\"
```

Comment:
You should export this to config.h.

Reply:
What's written there does not apply, `config.h` defines using `AC_DEFINE(IS_BIG_ENDIAN, 1, Big endian)` is more flexible than CPPFLAGS.

Reply:
`AX_DEFINE_DIR` and `AX_DEFINE_SUB_PATH` are broken.

https://github.com/autoconf-archive/autoconf-archive/blob/dcff834afae34861f7f11f3d51e0c938a35bd30f/m4/ax_define_sub_path.m4#L7

At least I couldn't get them to work.

Reply:
```
  --datarootdir=DIR       read-only arch.-independent data root [PREFIX/share]
  --datadir=DIR           read-only architecture-independent data [DATAROOTDIR]
  --infodir=DIR           info documentation [DATAROOTDIR/info]
  --localedir=DIR         locale-dependent data [DATAROOTDIR/locale]
  --mandir=DIR            man documentation [DATAROOTDIR/man]
  --docdir=DIR            documentation root [DATAROOTDIR/doc/libtorrent]
```

There are already flags for controlling the data dir.